homeassistant/components/unifi/sensor.py

```diff
@@ -169,6 +178,11 @@ class UnifiSensorEntityDescription(
     """Class describing UniFi sensor entity."""
 
     is_connected_fn: Callable[[UniFiController, str], bool] | None = None
+    # Custom function to determine whether a state change should be recorded
+    value_changed_fn: Callable[
```

Comment:
Maybe call this significant_value_change

Reply:
In an earlier draft, this was `value_change_threshold_fn` or `value_noise_threshold_fn` :shrug: There's probably prior art in other core integrations but I didn't spent a ton of effort on digging.